tools/packaging/kata-deploy/runtimeclasses/kata-qemu-tdx.yaml

```diff
@@ -8,6 +8,7 @@ overhead:
     podFixed:
         memory: "2048Mi"
         cpu: "1.0"
+        #tdx.intel.io/keys: "1"
```

Comment:
This is intemtional, and we only want to uncomment tis in case we're using NFD, which is done here: https://github.com/kata-containers/kata-containers/pull/7469/files#diff-08a680425cfe93e69c4108dc8fd91a9aeaefbf016bd9941b03c66834bf87a341R55

Reply:
Oh cool. I missed your sed wizardry. 🧙